scripts/setup.sh

```diff
@@ -56,19 +56,48 @@ setup_haskell() {
   ln -sf ${STACK_BIN_PATH} ${HVN_DEST}/.stack-bin
 
   msg "Installing helper binaries..."
-  local STACK_LIST="ghc-mod hlint hasktags hscope pointfree pointful hoogle apply-refact machines-directory-0.2.0.9 machines-io-0.2.0.13 codex-0.5.0.2"
-  stack --resolver ${STACK_RESOLVER} install ${STACK_LIST} --verbosity warning ; RETCODE=$?
-  [ ${RETCODE} -ne 0 ] && exit_err "Binary installation failed with error ${RETCODE}."
-
-  # Install hindent 5 via pinned nightly
-  stack --resolver nightly-2016-12-28 install hindent --verbosity warning ; RETCODE=$?
-  [ ${RETCODE} -ne 0 ] && exit_err "hindent installation failed with error ${RETCODE}."
+  # Install ghc-mod via active stack resolver for maximum out-of-the-box compatibility.
+  stack --resolver ${STACK_RESOLVER} install ghc-mod --verbosity warning ; RETCODE=$?
+  [ ${RETCODE} -ne 0 ] && exit_err "Installing ghc-mod failed with error ${RETCODE}."
+
+  # Install hindent via pinned LTS to ensure we have version 5.
```

Comment:
Yes, it's a little frustrating that we have to manually install `hindent`.  The vim config expects hindent 5 and I'd rather not regress to hindent 4 to match the solver version.